lib/utils/search.js

```diff
@@ -1,6 +1,12 @@
 import { normalise } from './normalise';
 import { removeCommonWords, getSearchableFields } from 'lib/utils/searchHelper';
 
+const filterByCategories = (ids, items) => {
+  if (!ids || !ids.length) return items;
```

Comment:
is `!ids.length` the same as `ids.length<1`?

Reply:
Yep if you run this it will demonstrate: 
if (![].length) alert('1');